src/java.base/share/classes/jdk/internal/icu/impl/StringPrepDataReader.java

```diff
@@ -119,7 +119,7 @@ public byte[] getUnicodeVersion(){
      * No guarantees are made if a older version is used
      * see store.c of gennorm for more information and values
      */
-    ///* dataFormat="SPRP" 0x53, 0x50, 0x52, 0x50  */
+    // /* dataFormat="SPRP" 0x53, 0x50, 0x52, 0x50  */
```

Comment:
This source file is coming from the upstream ICU4J project. Even if this is a `non-standard` comment, I would keep it as it is to minimize the merge effort.

Reply:
As a non-standard comment, it will trigger a warning, since the prevailing standard for `java.base` is to compile with all warnings enabled and no warnings found.

Reply:
OK, we will need to live with it.

Reply:
Yes, while there is a strong preference to leave upstream sources unedited in the JDK, to my mind being able to compile the java.base module with all warnings enabled should take precedence.